app/views/course/achievements/show.html.slim

```diff
@@ -0,0 +1,15 @@
+- add_breadcrumb format_inline_text(@achievement.title)
+= page_header format_inline_text(@achievement.title)
+
+= render partial: 'achievements',
+         locals: { include_requirements: false, achievements: [@achievement] }
+
+div.user-achievements
+  h2 = t('.secondary_header')
```

Comment:
I would prefer some key that evokes an idea of what does section does. Maybe users_with_badge